api/soap/mantisconnect.wsdl

```diff
@@ -163,6 +164,8 @@
    <xsd:element name="monitors" type="tns:AccountDataArray" minOccurs="0"/>
    <xsd:element name="sticky" type="xsd:boolean" minOccurs="0"/>
    <xsd:element name="tags" type="tns:ObjectRefArray" minOccurs="0"/>
+   <xsd:element name="alias" type="xsd:string" minOccurs="0"/>
```

Comment:
Alias is really a concept that needs to only be understood by the report API and the app should be able to generate it on its own.  There is no benefit of returning this as part of the issue information as part of our public API.